docs/kubernetes/how-to-migrate-from-k8s-alpha-to-terraform/index.md

```diff
@@ -0,0 +1,257 @@
+external_resources:
+- '[Kubernetes Concepts Documentation](https://kubernetes.io/docs/concepts/)'
+- '[Beginners Guide to Terraform](https://www.linode.com/docs/applications/configuration-management/beginners-guide-to-terraform)'
+- '[Using Terraform to Provision Linode Environments](https://www.linode.com/docs/applications/configuration-management/how-to-build-your-infrastructure-using-terraform-and-linode/)'
+---
+
+The [k8s-alpha CLI](https://www.linode.com/docs/kubernetes/how-to-deploy-kubernetes-on-linode-with-k8s-alpha-cli/) is deprecated. On **March 31st, 2020**, it will be **removed** from the [linode-cli](https://github.com/linode/linode-cli). After March 31, 2020, you will no longer be able to create or manage clusters created by the k8s-alpha CLI plugin, however, you will still be able to successfully manage your clusters using [Terraform](https://www.terraform.io/).
+
+## In This Guide
+You will use Terraform to continue to manage and support clusters created using the k8s-alpha CLI plugin following the EOL date and beyond. You will learn how to:
+
+- [Manage clusters](#manage-k8s-alpha-clusters) created with the k8s-alpha CLI following it's deprecation.
+- [Delete clusters](#delete-a-cluster) using Terraform.
+- [Create clusters](#create-a-cluster) using Terraform.
+
+## Manage k8s-alpha Clusters
+
+The k8s-alpha CLI plugin was based on Terraform. As a result, it created a number of Terraform configuration files whenever it created a cluster. These Terraform files are found within the `.k8s-alpha-linode` directory. You can change into this directory using the following syntax:
+
+    cd $HOME/.k8s-alpha-linode
+
+If you list the contents of this directory, you will see a subdirectory for each of the clusters you've created with the k8s-alpha CLI plugin. For any of your clusters, contents of these subdirectories will be as follows:
+
+{{< output >}}
+drwxr-xr-x  5 username  staff   160 Dec 11 08:10 .terraform
+-rw-r--r--  1 username  staff   705 Dec 11 08:10 cluster.tf
+-rw-r--r--  1 username  staff  5456 Dec 11 08:14 example-cluster.conf
+-rw-r--r--  1 username  staff  5488 Dec 11 08:16 example-cluster_new.conf
+drwxr-xr-x  3 username  staff    96 Dec 11 08:10 terraform.tfstate.d
+{{</ output >}}
+
+- Both of the `.conf` files are kubeconfig files for this cluster.
+- `terraform.tfstate.d` is a Terraform state directory.
+- `.terraform` is a hidden directory which contains Terraform configuration files.
+- `cluster.tf` is the Terraform module file. This is the most important file here because it will allow you to scale, upgrade, and delete your cluster.
+
+{{< note >}}
+For more information regarding these files and directories and their contents, see our [Beginner's Guide to Terraform](/applications/configuration-management/beginners-guide-to-terraform/)
+{{< /note >}}
+
+### Scale a Cluster
+
+1.  To scale your cluster, edit `cluster.tf` with the text editor of your choice. The contents will be similar to the following:
```

Comment:
This line has the same beginning as step 2: "To scale your cluster..." One should change, and I think it's this one.

Reply:
updated